### PR TITLE
Chapter 3 Storage

### PR DESCRIPTION
**Pull request by darienmt**

### CODE REVIEW

File: content/chapter-03/AzureCloudStorage.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## [Azure Cloud Storage](https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction)
+
+* [Disks](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview)
+  * Aims Virtual Machine deployment
+  * Two options: [Standard](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/standard-storage) and [Premium](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/premium-storage)
+  
+  
+* [Blob storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+  * Common uses include: Serving images, storing files, streaming video and audio
+  * Three variations: Block blobs, Append blobs, and page blobs
```
Comment (vkhazin) [inline content/chapter-03/AzureCloudStorage.md]: href links would be nice for each variation

File: content/chapter-03/AzureCloudStorage.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  
+  
+* [Azure Files](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction)
+  * Replace or supplement traditional on-premises file servers or NAS
```
Comment (vkhazin) [inline content/chapter-03/AzureCloudStorage.md]: file servers, NAS, or SAN

File: content/chapter-03/OracleCloudStorage.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Storage space and IO speed
+
+## [Oracle Cloud Storage options](https://cloud.oracle.com/en_US/infrastructure/storage)
+
+* [Oracle Cloud Infrastructure Block Volumes](https://docs.us-phoenix-1.oraclecloud.com/Content/Block/Concepts/overview.htm)
+  * High Performance
+  * Boot Volumes
+  * Easily Scale Up or Down
```
Comment (vkhazin) [inline content/chapter-03/OracleCloudStorage.md]: Easily Scales Up or Down

**Review by vkhazin — CHANGES_REQUESTED** [review]

File: content/chapter-03/AzureCloudStorage.md (hunk 2 above)
Comment (darienmt) [reply content/chapter-03/AzureCloudStorage.md]: Done

File: content/chapter-03/OracleCloudStorage.md (hunk 3 above)
Comment (darienmt) [reply content/chapter-03/OracleCloudStorage.md]: Done

File: content/chapter-03/AzureCloudStorage.md (hunk 1 above)
Comment (darienmt) [reply content/chapter-03/AzureCloudStorage.md]: References added.

**Pull request merged**